src/USNewsletterEpic/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { css, ThemeProvider } from '@emotion/react';
+import React, { ReactElement } from 'react';
+import { brand } from '@guardian/src-foundations';
+import { Button, buttonBrandAlt } from '@guardian/src-button';
+import { styles as commonStyles } from '../styles/bannerCommon';
+import { COMPONENT_NAME } from './canRender';
+import { textSans } from '@guardian/src-foundations/typography';
+import { canRender } from './canRender';
+import { OphanComponentEvent } from '@guardian/types';
+import { BrazeClickHandler } from '../utils/tracking';
+import { until } from '@guardian/src-foundations/mq';
+
+// Once https://github.com/guardian/source/pull/843 is merged and in a
+// @guardian/src-icons release we'll be able to bump the version on this project
+// and replace this with:
+// import { SvgClock } from '@guardian/src-icons'
+export const SvgClock = (): ReactElement => {
+    return (
+        <svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+            <path
+                fillRule="evenodd"
+                clipRule="evenodd"
+                d="M26 15c0 6.075-4.925 11-11 11S4 21.075 4 15 8.925 4 15 4s11 4.925 11 11zm-9.8-.35L15.475 6H14.5l-.75 9.375 1.275 1.275L22 16v-1l-5.8-.35z"
+            />
+        </svg>
+    );
+};
+
+const IMAGE_URL =
+    'https://i.guim.co.uk/img/media/d0944e021b1cc7426f515fecc8034f12b7862041/0_0_784_784/784.png?width=196&s=fbdead3f454e1ceeeab260ffde71100a';
```

Comment:
A thing I learned from talking to folks who work on the Grid recently is that we should always be using the `master` crop as the source image and letting the image service do any resizing. Let's update this accordingly.